samples/corda/fungible-house-token/README.md

```diff
@@ -1,6 +1,6 @@
 # Fungible House Token
 
-The source is obtained from corda samples-kotlin repo: [FungibleHouseTokens](https://github.com/corda/samples-kotlin/tree/master/Tokens/fungiblehousetoken).
+The source is obtained from corda samples-kotlin repo: [Samples-Kotlin](https://github.com/corda/samples-kotlin), in directory `Tokens/fungiblehousetoken`.
```

Comment:
Rephrase: "The source is obtained from the `Tokens/fungiblehousetoken` directory in the Corda `samples-kotlin` repository: [Samples-Kotlin](https://github.com/corda/samples-kotlin)."